src/django/api/views.py

```diff
@@ -608,7 +621,22 @@ def claim(self, request, pk=None):
 
             send_claim_facility_confirmation_email(request, facility_claim)
 
-            return Response(status=status.HTTP_204_NO_CONTENT)
+            approved = FacilityClaim \
+                .objects \
+                .filter(status=FacilityClaim.APPROVED) \
+                .filter(contributor=contributor) \
+                .values_list('facility__id', flat=True)
+
+            pending = FacilityClaim \
+                .objects \
+                .filter(status=FacilityClaim.PENDING) \
+                .filter(contributor=contributor) \
+                .values_list('facility__id', flat=True),
```

Comment:
We were bitten by the Python! There is a trailing comma hiding at the end of this line which ended up creating a tuple. I was able to remove that comma and the `list(chain(*))` wrappers and see things working as expected.

Reply:
Thanks for catching that! I fixed this in amended commit a6dd71f, along with the instance in the serializers file.